api/validators/MerchStoreRequests.ts

```diff
@@ -258,6 +262,10 @@ export class OrderPickupEvent implements IOrderPickupEvent {
   @IsDefined()
   @Min(1)
   orderLimit: number;
+
+  @IsOptional()
+  @IsUUID()
+  linkedEvent?: EventModel;
```

Comment:
Validating this with type EventModel doesn't seem correct to me although it satisfies any type errors, is this validation even necessary for this optional field?

Reply:
yeah i suppose just for the possibility of storing non-uuid stuff in the database